core/gesture.ts

```diff
@@ -691,8 +700,6 @@ export class Gesture {
 
     if (this.isPinchZoomEnabled && this.cachedPoints.size === 2) {
       this.handlePinch(e);
-    } else {
-      this.handleMove(e);
```

Comment:
I think we still want to handle the move in the case where pinch to zoom is disabled! Could you add a check to this else that `this.cachedPoints.size < 3` so if there are 1 or two fingers, we still perform the drag?

Reply:
Hi, for me the "handleMove" have nothing to do here in any case. Because if it's called, it automatically generates an infinite loop because "handleTouchMove" is called by "handleMove"... Here's an example with pinch zoom disabled as you proposed.

1. First finger down
    a. Calls "BindMouseEvents" => Making the next finger to call "handleStart"
    b. Calls "HandleTouchStart" => Register the finger in "cachedPoints"
2. Second finger down (without moving the first)
    a. Calls "handleStart" => (called because of the bindMouse call previously)
    b. Calls "HandleTouchStart" => Register the finger in "cachedPoints" and set isMultiTouch_ = true
3. Move a finger
    a. Calls "handleMove" (because of BindMouseEvents)
    b. Calls "handleTouchMove" because we are in multi touch and not dragging
    c. In your case (this.cachedPoints.size < 3 is true) so it calls "handleMove"
    d. In "handleMove" we have the same conditions as before, the multi touch condition is executed, it calls "handleMultiTouch" and so on... Infinite loop